chromium/pages/options/ux.js

```diff
@@ -329,14 +329,29 @@ document.addEventListener("DOMContentLoaded", () => {
 
   const add_disabled_site = document.getElementById("add-disabled-site");
   const disabled_site_input = document.getElementById("disabled-site");
+  const add_disabled_site_invalid_host = document.getElementById('add-disabled-site-invalid-host');
   disabled_site_input.setAttribute("placeholder", chrome.i18n.getMessage("options_enterDisabledSite"));
-
+function isValidHost(host) {
+  try {
+    new URL(`http://${host}/`);
+    return true;
+  } catch {
+    return false;
+  }
+}
```

Comment:
Please indent this function properly:

```suggestion
  function isValidHost(host) {
    try {
      new URL(`http://${host}/`);
      return true;
    } catch {
      return false;
    }
  }
```